cypress/e2e/transformation.cy.js

```diff
@@ -1,4 +1,8 @@
 describe('Report transformation', () => {
+  before(() => {
+    cy.resetApp();
```

Comment:
Moet dit niet in beforeEach()? En dan kan cy.clearDebugStore() denk ik weg uit beforeEach(), klopt dat?

Reply:
Om de testen sneller en meer stabiel te running wordt de resetApp alleen in een before gedaan. In resetApp worden veel dingen gedaan die eigenlijk niet handig zijn als die na elke test uitgevoerd worden.
In dit geval is het alleen nodig om de clearDebugStore() aan te roepen voor elke test. Dit komt ook nog terug in een paar andere testen waar dit nodig was.

Reply:
Ok.